tests/cloud/test_service_id.py

```diff
@@ -162,3 +172,70 @@ def test_no_error_raised_if_service_sruid_components_valid(self):
         ):
             with self.subTest(namespace=namespace, name=name, revision_tag=revision_tag):
                 validate_service_sruid(namespace=namespace, name=name, revision_tag=revision_tag)
+
+
+class TestSplitServiceID(unittest.TestCase):
+    def test_split_sruid(self):
+        """Test that a valid SRUID can be split into its namespace, name, and revision tag."""
+        namespace, name, revision_tag = split_service_id("octue/my-service:2.3.0")
```

Comment:
Add tests for invalid values like `octue/my-service:`, `myservice:1.2.0`, `/myservice`, `/myservice:` and `myservice`. I commented above that `octue/my-service:` should return `None` as a revision tag, but actually I think all of these should raise an exception inheriting from ValueError